CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.5 - TBD
+
+### Fixed
+- Fixed command for update start and end dates when using timezone.
```

Comment:
Could you please rephrase this sentence? I think it's not really describing what was fixed and where.